.docker/Dockerfile.cli.onlytheme

```diff
@@ -0,0 +1,111 @@
+COPY patches /app/patches
+COPY scripts /app/scripts
+
+RUN mkdir -p docroot/themes/contrib/civictheme \
+    && mkdir -p docroot/themes/contrib/civictheme \
+    && mkdir -p docroot/modules/custom/civictheme_govcms \
+    && mkdir -p docroot/modules/custom/civictheme_admin \
+    && mkdir -p docroot/modules/custom/civictheme_migrate \
+    && mkdir -p docroot/modules/custom/civictheme_content \
+    && mkdir -p docroot/modules/custom/civictheme_dev \
+    && mkdir -p docroot/modules/custom/civictheme_styleguide \
+    && mkdir -p docroot/modules/custom/cs_generated_content
+
+# Copy files required for PHP dependencies resolution.
+# Note that composer.lock is not explicitly copied, allowing to run the stack
+# without existing lock file (this is not advisable, but allows to build
+# using latest versions of packages). composer.lock should be comitted to the
+# repository.
+# File .env (and other environment files) is copied into image as it may be
+# required by composer scripts to get some additions variables.
+COPY composer.json composer.* .env* auth* /app/
+
+COPY docroot/themes/contrib/civictheme/composer.json /app/docroot/themes/contrib/civictheme/
+COPY docroot/modules/custom/civictheme_govcms/composer.json docroot/modules/custom/civictheme_govcms/
+COPY docroot/modules/custom/civictheme_admin/composer.json docroot/modules/custom/civictheme_admin/
+COPY docroot/modules/custom/civictheme_migrate/composer.json docroot/modules/custom/civictheme_migrate/
+COPY docroot/modules/custom/civictheme_content/composer.json docroot/modules/custom/civictheme_content/
+COPY docroot/modules/custom/civictheme_dev/composer.json docroot/modules/custom/civictheme_dev/
+COPY docroot/modules/custom/civictheme_styleguide/composer.json docroot/modules/custom/civictheme_styleguide/
+COPY docroot/modules/custom/cs_generated_content/composer.json docroot/modules/custom/cs_generated_content/
+
+# Install PHP dependencies, but without development dependencies. This is very
+# important, because we do not want potential security issues to be exposed to
+# production environment.
+RUN COMPOSER_MEMORY_LIMIT=-1 composer install -n --no-dev --ansi --prefer-dist --optimize-autoloader
+
+# Install NodeJS dependencies.
+# Note that package-lock.json is not explicitly copied, allowing to run the
+# stack without existing lock file (this is not advisable, but allows to build
+# using latest versions of packages). package-lock.json should be comitted to
+# the repository.
+# File Gruntfile.sj is copied into image as it is required to generate
+# front-end assets.
+COPY docroot/themes/contrib/civictheme/ docroot/themes/contrib/civictheme/package* /app/docroot/themes/contrib/civictheme/
+
+# Install NodeJS dependencies.
+# Since Drupal does not use NodeJS for production, it does not matter if we
+# install development dependencies here - they are not exposed in any way.
+RUN npm --prefix docroot/themes/contrib/civictheme install --no-audit --no-progress --unsafe-perm
```

Comment:
So no longer running npm install on civictheme_library is the big change?

Reply:
we are still building the library as a part of `civictheme` Drupal theme. The only time we would need to build the library in CI is for publishing it to a standalone repo.

From the point of build failure - if library has an issue - the build would fail as the issue would be discovered in `civictheme`.